src/explore-education-statistics-admin/src/pages/bau/BauImportsPage.tsx

```diff
@@ -0,0 +1,94 @@
+    importStatusBauService.getAllIncompleteImports(),
+  );
+
+  return (
+    <Page
+      wide
+      breadcrumbs={[
+        { name: 'Platform administration', link: '/administration' },
+        { name: 'Incomplete imports' },
+      ]}
+    >
+      <h1 className="govuk-heading-xl">
+        <span className="govuk-caption-xl">Manage incomplete imports</span>
+        Incomplete imports
+      </h1>
+      <table className="govuk-table">
+        <thead className="govuk-table__head">
+          <tr className="govuk-table__row">
+            {/* EES-1655 */}
+            {/* <th scope="col" className="govuk-table__header">
+              Subject title
+            </th> */}
+            <th scope="col" className="govuk-table__header">
+              Subject ID
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Data file name
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Meta file name
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Data file rows
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Status
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Stage Percentage Complete
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Go to release
+            </th>
+          </tr>
+        </thead>
+        <LoadingSpinner loading={isLoading} text="Loading imports">
+          {value && (
+            <tbody className="govuk-table__body">
+              {value.map(subject => (
+                <tr className="govuk-table__row" key={subject.subjectId}>
+                  {/* EES-1655 */}
+                  {/* <th scope="row" className="govuk-table__header">
+                    {subject.subjectTitle}
+                  </th> */}
+                  <td className="govuk-table__row">{subject.subjectId}</td>
+                  <td className="govuk-table__row">{subject.dataFileName}</td>
+                  <td className="govuk-table__row">{subject.metaFileName}</td>
+                  <td className="govuk-table__row">{subject.numberOfRows}</td>
+                  <td className="govuk-table__row">{subject.status}</td>
+                  <td className="govuk-table__row">
+                    {subject.stagePercentageComplete}
+                  </td>
+                  <td className="govuk-table__row">
+                    <Link
+                      to={`/publication/${subject.publicationId}/release/${subject.releaseId}/data`}
+                    >
+                      Go to release
+                    </Link>
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          )}
+        </LoadingSpinner>
+      </table>
+      <p>
+        <Link to="/administration/" className="govuk-back-link">
```

Comment:
Use the `back` prop instead of the class.

Reply:
I've removed it - the `backLink`/`backLinkText` Page props put the link at the top of the page, and that seemed superfluous given the breadcrumb was immediately above that, so I thought it was better to remove it.